Makefile

```diff
@@ -49,3 +49,28 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+DURATION ?= 3s
+COUNT ?= 5
+
+.PHONY: bench
+bench:
+	rm -rf bench
+	mkdir bench
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=${DURATION} -count=${COUNT} 2>&1 | tee bench/bench.txt
+	cd bench && git clone https://go.googlesource.com/perf && cd perf && go install ./cmd/benchstat
+	benchstat bench/bench.txt
+	rm -rf bench/perf
+
+.PHONY: perf-diff
+perf-diff:
+	rm -rf bench
+	mkdir bench
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=${DURATION} -count=${COUNT} 2>&1 | tee bench/new.txt
+	cd bench && git clone https://github.com/tarantool/go-tarantool.git
+	cd bench/go-tarantool && go test -bench=. -benchmem -benchtime=${DURATION} -count=${COUNT} 2>&1 | tee ../old.txt
+	cd bench && git clone https://go.googlesource.com/perf && cd perf && go install ./cmd/benchstat
+	benchstat bench/old.txt bench/new.txt
+	rm -rf bench/go-tarantool && rm -rf bench/perf
```

Comment:
`rm -rf bench/go-tarantool bench/perf`?

Reply:
Just for clearing the directory from unnecessary artifacts.

Reply:
I mean that you can execute `rm` only once and pass two directories at once.

Reply:
seems comment is not valid anymore